packages/flutter/lib/src/services/text_input.dart

```diff
@@ -182,7 +197,7 @@ class TextInputType {
   static const TextInputType streetAddress = TextInputType._(9);
 
   /// Prevent the OS from showing the on-screen virtual keyboard.
```

Comment:
Consider adding a comment similar to how `TextInputType number` has in this file.

```
  /// Additional options, such as decimal point and/or positive/negative
  /// signs, can be requested using [TextInputType.numberWithOptions].
```